.github/workflows/scripts/run_build_ubuntu.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+set -e
+
+sudo apt update
+sudo apt install cmake clang libprotobuf-dev protobuf-compiler
+python -m pip install --upgrade pip
+pip install -r requirements_dev.txt
```

Comment:
you can build the library here too/run the tests

Reply:
ok

Reply:
can you give an example, like how can I segregate the commands in the same file